cypress-tests/cypress/support/commands.js

```diff
@@ -484,16 +545,17 @@ Cypress.Commands.add("citForMandatesCallTest", (requestBody, amount, details, co
       "Content-Type": "application/json",
       "api-key": globalState.get("apiKey"),
     },
+    failOnStatusCode: false,
     body: requestBody,
   }).then((response) => {
     logRequestId(response.headers['x-request-id']);
-
+    expect(res_data.status).to.equal(response.status);
     expect(response.headers["content-type"]).to.include("application/json");
-    expect(response.body).to.have.property("mandate_id");
     globalState.set("mandateId", response.body.mandate_id);
     globalState.set("paymentID", response.body.payment_id);
 
     if (response.body.capture_method === "automatic") {
```

Comment:
same here. can we have `if` statement with http status 2xx or 4xx condition for better readability